frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/default-template.ts

```diff
@@ -28,7 +27,7 @@ export const resolveDefaultVMTemplate = (params: DefaultVMLikeEntityParams): Tem
   const vm = template.getVM();
   const containerDiskName = 'containerdisk';
 
-  vm.setHostname(VM_TEMPLATE_NAME_PARAMETER)
+  vm.setHostname(vm.getName())
```

Comment:
we want to keep this, this is still useful.

Reply:
My fault. I misunderstood our conversation and thought we were eliminating it entirely. Fixed. =)